client/src/main/java/com/tencent/rss/client/impl/ShuffleReadClientImpl.java

```diff
@@ -80,6 +78,19 @@ public ShuffleReadClientImpl(
     this.partitionId = partitionId;
     this.blockIdBitmap = blockIdBitmap;
     this.taskIdBitmap = taskIdBitmap;
+
+    List<Long> removeBlockIds = Lists.newArrayList();
+    blockIdBitmap.forEach(bid -> {
```

Comment:
why change the code‘s position？

Reply:
The position shouldn't be changed, I want to pass the bitmap to handler to get index file size, change the position may get wrong file size. I will fix it, thanks.